src/lib/maven.js

```diff
@@ -79,3 +81,11 @@ export function downloadArtifact(artifact, urlDownloadTemplate, verbose) {
       });
   });
 }
+
+export async function installToLocalRepository(archive, pomFile) {
```

Comment:
It is good, however, by removing `repoPath`, we are no longer able to install file without having `mvn` command :(.
However, I'm not sure if that was really usable.